### ISSUE TITLE
No headers raises a KeyError

### ISSUE THREAD

**Issue by jones-chris** [slank/awsgi#41]
When the headers key has a value of None, a NoneType error is thrown on line 129 of __init__.py.  Here is an example stacktrace:

```
[ERROR] AttributeError: 'NoneType' object has no attribute 'items'
Traceback (most recent call last):
  File "/var/task/lambda_function.py", line 79, in lambda_handler
    return awsgi.response(app, event, context)
  File "/var/task/awsgi/__init__.py", line 159, in response
    output = app(environ(event, context), sr)
  File "/var/task/awsgi/__init__.py", line 129, in environ
    for k, v in headers.items():
```

After looking at the code, I think it can be fixed by only entering line 129 (the for loop) if items is not None or has a non-zero length.

Please let me know if you have any questions or need more information.

**Comment by slank** [comment]
Hi, @jones-chris. Thanks for the report. I think this is worth fixing because it improves the robustness of the code.

I'm curious though: when, in practice, would this condition occur?

**Comment by jones-chris** [comment]
Thanks @slank !  I came across this today when I was setting up an API Gateway endpoint to call a Flask lambda.  I tested the endpoint using the API Gateway GUI and did not add any headers because none were required by the Flask lambda at the stage I was at in development.  Adding an "Accept:  application/json" worked, but was unnecessary in my case.

I've never used the awsgi before, so I was confused by the error.  I appreciate you fixing it - it'll be good for those who are new to awsgi like myself and for testing when headers may not be used 😄 .
